src/reducers/mapReducer.js

```diff
@@ -0,0 +1,24 @@
+import Immutable from 'immutable';
+import * as types from '../actions/actionTypes';
+
+export const initialState = Immutable.Map({
```

Comment:
Destructure Immutable when importing.
```suggestion
export const initialState = Map({
```